tests/golos.vesting_tests.cpp

```diff
@@ -44,14 +45,19 @@ class golos_vesting_tester : public golos_tester {
         BOOST_CHECK_EQUAL(success(), token.create(cfg::emission_name, token.make_asset(supply), {cfg::charge_name, cfg::publish_name}));
         BOOST_CHECK_EQUAL(success(), token.issue(cfg::emission_name, N(sania), token.make_asset(issue1), "issue tokens sania"));
         BOOST_CHECK_EQUAL(success(), token.issue(cfg::emission_name, N(pasha), token.make_asset(issue2), "issue tokens pasha"));
+        produce_block();
+
+        BOOST_CHECK_EQUAL(err.not_found_token_vesting, vest.open(N(sania)));
 
+        BOOST_CHECK_EQUAL(success(), vest.create_vesting(cfg::emission_name));
         BOOST_CHECK_EQUAL(success(), vest.open(N(sania)));
         BOOST_CHECK_EQUAL(success(), vest.open(N(pasha)));
         BOOST_CHECK_EQUAL(success(), vest.open(N(tania)));
         BOOST_CHECK_EQUAL(success(), vest.open(N(vania)));
+
+        BOOST_CHECK_EQUAL(err.mismatch_of_accuracy, vest.open(N(sania), _vesting_sym_e, N(sania)));
         produce_block();
 
-        BOOST_CHECK_EQUAL(success(), vest.create_vesting(cfg::emission_name));
```

Comment:
It's not needed to move `create_vesting` from here If you want check missing vesting case. You can pass another vesting name in `open` function.